dashboard/app/controllers/api/v1/pd/applications_controller.rb

```diff
@@ -71,11 +71,22 @@ def quick_view
   # GET /api/v1/pd/applications/cohort_view?role=:role&regional_partner_filter=:regional_partner_name
```

Comment:
To me this doesn't really belong with applications_controller. Regional partner capacity is a property of the regional partner, so this seems like it belongs more in the regional partner controller.

Given the time of this, it may not be feasible to change this.